website/docs/r/service_dictionary_items_v1.html.markdown

```diff
@@ -0,0 +1,212 @@
+---
+layout: "fastly"
+page_title: "Fastly: service_dictionary_items_v1"
+sidebar_current: "docs-fastly-resource-service-v1"
+description: |-
+  Provides a grouping of fastly dictionary items that can be applied to a service. 
+---
+
+# fastly_service_dictionary_items_v1
+
+Provides a grouping of fastly dictionary items that can be applied to a service.
+ 
+This resource will populate a dictionary with the items and it will track their state going forward.
+Additional dictionary items can be added through the Fastly API/console, but will be removed from fastly if terraform detects a difference in the remote state.
+
+The Fastly API/console can also be used to modify the items that are managed through Terraform.  In this case the default behaviour of the 
```

Comment:
Please change "API/console" to "API or UI"

Reply:
Files were searched and updated where appropriate.